media/js/firefox/developer/whatsnew-mdnplus.js

```diff
@@ -0,0 +1,52 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+(function () {
+    'use strict';
+
+    var hidden;
+    var visibilityChange;
+    var mandalaElement = document.getElementById('mandala');
+
+    // Check for support of the Page Visbility API
+    if (typeof document.hidden !== 'undefined') {
+        hidden = 'hidden';
+        visibilityChange = 'visibilitychange';
+    } else {
+        return;
+    }
+
+    // If the page is hidden, stop the mandala.
+    // If the page is visible, animate the mandala.
+    function handleVisibilityChange() {
+        if (document[hidden]) {
```

Comment:
Nit: suggest simplifying to `document.hidden`
```suggestion
        if (document.hidden) {
```